modules/nf-core/manta/tumoronly/main.nf

```diff
@@ -3,15 +3,16 @@ process MANTA_TUMORONLY {
     label 'process_medium'
     label 'error_retry'
 
-    conda "bioconda::manta=1.6.0"
+    conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/manta:1.6.0--h9ee0642_1' :
         'biocontainers/manta:1.6.0--h9ee0642_1' }"
 
     input:
     tuple val(meta), path(input), path(input_index), path(target_bed), path(target_bed_tbi)
-    path fasta
-    path fai
+    tuple val(meta2), path(fasta)
```

Comment:
Can you double check that nothing on the input channels needs adapting. I would expect the usual errors and cardinality complaints

Reply:
Done already